scripts/debcheck.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Should it use `set -e` to fail early?
```suggestion
#!/bin/bash

set -e
```

Reply:
and also pipefail, yes!